client/state/selectors/is-woo-site.js

```diff
@@ -0,0 +1,19 @@
+import { get } from 'lodash';
+import { isJetpackSite } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+
+import 'calypso/state/plugins/init';
+
+export default ( state ) => {
+	const siteId = getSelectedSiteId( state );
+
+	if (
+		isJetpackSite( state, siteId ) &&
+		// @todo This should be updated to be more specific to Woo.com sites.
+		get( state, [ 'sites', 'items', siteId, 'options', 'woocommerce_is_active' ], null )
```

Comment:
Use the `getSiteOption( state, siteId, 'woocommerce_is_active' )` selector here.